cloud/amazon/route53_zone.py

```diff
@@ -54,25 +54,34 @@
 
 EXAMPLES = '''
 # create a public zone
-- route53_zone: zone=example.com state=present comment="this is an example"
+- route53_zone:
+    zone: example.com
+    state: present
+    comment: this is an example
 
 # delete a public zone
-- route53_zone: zone=example.com state=absent
+- route53_zone:
+    zone: example.com
+    state: absent
 
 - name: private zone for devel
-  route53_zone: zone=devel.example.com state=present vpc_id={{myvpc_id}} comment='developer domain'
+  route53_zone:
+    zone: devel.example.com
+    state: present
+    vpc_id: '{{ myvpc_id }}'
+    comment: developer domain
 
 # more complex example
 - name: register output after creating zone in parameterized region
   route53_zone:
-    vpc_id: "{{ vpc.vpc_id }}"
-    vpc_region: "{{ ec2_region }}"
-    zone: "{{ vpc_dns_zone }}"
+    vpc_id: '{{ vpc.vpc_id }}'
+    vpc_region: '{{ ec2_region }}'
+    zone: '{{ vpc_dns_zone }}'
     state: present
     register: zone_out
 
-- debug: var=zone_out
-
+- debug:
+    var: zone_out
```

Comment:
I'd need to check, though I think this should have quotes and double curly braces.